topcoffea/modules/event_selection.py

```diff
@@ -46,7 +48,10 @@ def trg_pass_no_overlap(events,is_data,dataset,year,dataset_dict,exclude_dict):
     trg_passes = passes_trg_inlst(events,full_trg_lst)
 
     # In case of data, check if events overlap with other datasets
-    if is_data:
+    if is_data and (era is not None):   #Used for potential era dependency in Run3
```

Comment:
I think this would be easier to read if the logic looked like this:
```
if is_data:
    if era is not none:
        do stuff
    else:
         do other stuff
```